app/controllers/sitemaps_controller.rb

```diff
@@ -1,6 +1,12 @@
 class SitemapsController < ApplicationController
```

Comment:
How often are the sitemap pages accessed? I assume web crawlers check them, but is that often or fairly irregularly?

Reply:
Often. I imagine multiple times per day, and the sql query takes about 40secs.

Reply:
Okay. I was wondering whether it would be a good idea to cache, but a couple of times a day is not worth it I think.